ios/FluentUI/Drawer/DrawerShadowView.swift

```diff
@@ -71,27 +63,20 @@ class DrawerShadowView: UIView {
         owner = nil
     }
 
-    func animate(withDuration duration: TimeInterval, animations: () -> Void) {
```

Comment:
Was this just not being called anywhere? Is that a mistake? I.e., was the animate() call accidentally removed in a merge somehow? Just making sure we're removing functionality intentionally.

Reply:
It wasn't called anywhere and I couldn't find any calls to it in the file history either. I think whoever wrote the script just used it during development and forgot to remove it.